distribution/ddf-common/src/main/resources/etc/blacklisted.properties

```diff
@@ -1,12 +1,12 @@
-pax-jetty;type=feature;range=[9.3,9.3.21]
+pax-jetty;type=feature;range=[9.3,9.4.10]
 pax-jetty-http2;type=feature
-pax-http-whiteboard;type=feature;range=[6.0.6,6.0.11]
+pax-http-whiteboard;type=feature;range=[7.0.0,7.2.3]
 
-pax-http-jetty;type=feature;range=[6.0.6,6.0.10]
-pax-http;type=feature;range=[6.0.6,6.0.10]
-pax-war;type=feature;range=[6.0.6,6.0.10]
-pax-http-tomcat;type=feature;range=[6.0.6,6.0.10]
-pax-war-tomcat;type=feature;range=[6.0.6,6.0.10]
-pax-jsf-support;type=feature;range=[6.0.6,6.0.10]
-pax-jsf-resources-support;type=feature;range=[6.0.6,6.0.10]
-pax-http-undertow;type=feature;range=[6.0.6,6.0.10]
+pax-http-jetty;type=feature;range=[7.0.0,7.2.2]
+pax-http;type=feature;range=[7.0.0,7.2.2]
+pax-war;type=feature;range=[7.0.0,7.2.2]
+pax-http-tomcat;type=feature;range=[7.0.0,7.2.2]
+pax-war-tomcat;type=feature;range=[7.0.0,7.2.2]
+pax-jsf-support;type=feature;range=[7.0.0,7.2.2]
+pax-jsf-resources-support;type=feature;range=[7.0.0,7.2.2]
+pax-http-undertow;type=feature;range=[7.0.0,7.2.2]
```

Comment:
I'm not sure if this affects 4.2 or a later version of karaf but blacklisted.properties has been deprecated.
https://issues.apache.org/jira/browse/KARAF-4181?focusedCommentId=16275591&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16275591

Reply:
👍 I'm looking to replace it with their new way of doing this. https://github.com/codice/ddf/pull/3635#discussion_r215085998

Reply:
DDF ticket created to resolve this: https://codice.atlassian.net/browse/DDF-4146